x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -340,4 +537,210 @@ public final void testSimpleConstantFolding() {
     public void testSerializationOfSimple() {
         assertSerialization(buildFieldExpression(testCase));
     }
+
+    /**
+     * Adds cases with {@code null} and asserts that the result is {@code null}.
+     */
+    protected static List<TestCaseSupplier> anyNullIsNull(boolean entirelyNullPreservesType, List<TestCaseSupplier> testCaseSuppliers) {
```

Comment:
nit: would be cool if the doc string explained the params a bit, esp. `entirelyNullPreservesType`.

Reply:
++. That was one I had to add later on to make everything make sense and I didn't doc it properly. Thanks for pointing it out!